tests/smoke/support/cta/files/misc.ts

```diff
@@ -21,8 +21,12 @@ export const resourceExists = async ({
   page: Page
   name: string
 }): Promise<boolean> => {
-  const resource = await page.$(`[data-test-resource-name="${name}"]`)
-  return !!resource
+  try {
+    await page.waitForSelector(`[data-test-resource-name="${name}"]`, { timeout: 100 })
```

Comment:
We made not-so-good experiences with setting explicit timeouts within the acceptance tests. Why should we re-introduce them here?